src/components/Dashboard/Balance.js

```diff
@@ -1,53 +1,173 @@
+  mainIcon,
+  mainIconBackground,
+  activity,
+  actions,
+}) {
+  const theme = useTheme()
+  const { anjToken } = useCourtConfig()
+
+  return (
+    <div>
       <div
         css={`
-          display: flex;
-          align-items: center;
+          border-bottom: 1px solid ${theme.border.alpha(0.7)};
         `}
       >
-        <img
+        <div
           css={`
-            margin-right: 8px;
-          `}
-          alt={symbol}
-          src={iconSrc}
-        />
-        <span
-          css={`
-            ${textStyle('label1')}
-            color: ${theme.contentSecondary};
+            display: flex;
+            align-items: flex-start;
+            padding-bottom: ${2 * GU}px;
           `}
         >
-          {symbol}
-        </span>
+          <div
+            css={`
+              padding: ${1.5 * GU}px;
+              background: ${mainIconBackground};
+              border-radius: 50%;
+              margin-right: ${2 * GU}px;
+            `}
+          >
+            <img
+              css={`
+                display: block;
+              `}
+              src={mainIcon}
+              height={3 * GU}
+              width={3 * GU}
+            />
+          </div>
+          <div>
+            <span
+              css={`      
+                ${textStyle('body2')}
+                color: ${theme.contentSecondary};
+                display:block;
+              `}
+            >
+              {label}
+            </span>
+            <div
+              css={`
+                ${textStyle('title3')}
+                line-height: 1.2;
+                display: flex;
+              `}
+            >
+              {splitAmount(amount.toFixed(2))} <img src={ANJIcon} />
+            </div>{' '}
+            <span
+              css={`
+                ${textStyle('body4')}
+                color: ${theme.contentSecondary};
+                display:block;
+              `}
+            >
+              $ {formatTokenAmount(convertedAmount.toFixed(2))}
+            </span>
+          </div>
+        </div>
       </div>
-      <div>
-        <span
+      <div
+        css={`
+          margin: ${2 * GU}px 0;
+          color: ${theme.contentSecondary};
+        `}
+      >
+        {activity ? (
+          <LatestActivity activity={activity} tokenSymbol={anjToken.symbol} />
+        ) : (
+          <span>No recent 24h activity</span>
+        )}
+      </div>
+
+      {amount > '0' && (
+        <div
           css={`
-            ${textStyle('title2')}
+            display: grid;
+            grid-template-columns: repeat(auto-fit, minmax(48%, 1fr));
+            grid-column-gap: 8px;
           `}
         >
-          {amount}
-        </span>
-      </div>
+          {actions.map((action, index) => {
+            return (
+              <Button
+                key={index}
+                label={action.label}
+                mode={action.mode}
+                onClick={action.onClick} // eslint-disable-line
```

Comment:
I think so, because didn't get any errors and the functionality worked as expected.
I deleted anyways, forgot to do it. 
If i recall correctly was having problems with a rule that didn't make sense so disabled it to not lose more time 😄